packages/create-sitecore-jss/src/templates/nextjs-personalize/src/lib/middleware/plugins/personalize.ts

```diff
@@ -2,6 +2,7 @@ import { NextRequest, NextResponse } from 'next/server';
 import { PersonalizeMiddleware } from '@sitecore-jss/sitecore-jss-nextjs/middleware';
 import { MiddlewarePlugin } from '..';
 import config from 'temp/config';
+import { PosResolver } from '../../pos-resolver';
```

Comment:
Can use "lib" alias here
```suggestion
import { PosResolver } from 'lib/pos-resolver';
```